main.go

```diff
@@ -54,5 +55,5 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseDevMode(false)))
 
-	controllers.Run(metricsAddr, enableLeaderElection, ctrl.GetConfigOrDie(), ctrl.SetupSignalHandler())
+	controllers.Run(metricsAddr, enableLeaderElection, ctrl.GetConfigOrDie(), ctrl.SetupSignalHandler(), !clowder_config.LoadedConfig.Webhooks.Disable)
```

Comment:
Any particular reason to do it this way rather than the `boolean` method from the mutating webhook?

Reply:
Not sure what you mean by this

Reply:
I guess what's the advantage of this vs [this other way](https://github.com/RedHatInsights/clowder/pull/292/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R57)

Reply:
Doing it the other means it is ALWAYS on, and was relying on an env var which didn't actually do anything - this new way is much safer and allows you to turn off webhooks when doing `make run`